README.md

```diff
@@ -154,9 +154,9 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/><br/>
 
-## ![✔] 2.5 Document API errors using Swagger
+## ![✔] 2.5 Document RESTful API errors using Swagger
```

Comment:
@BrunoScheufler Why not make it generic API advice:
"Document API errors using Swagger or graph"

Reply:
I think this is a bit more complicated from a language level because we either have REST APIs and Swagger or GraphQL APIs. Writing "Document API errors using Swagger or GraphQL" also sounds as if GraphQL was merely a tool to handle documentation of APIs, which it's clearly not. Not sure how to continue that

Reply:
GraphQL schema?
https://graphql.org/learn/schema/

Reply:
❓

Reply:
GraphQL schema is more or less the equivalent of REST swagger. It's always exposed to the client using https://graphql.org/learn/introspection/

In overall, since API techs vary (grpc, graph, etc) we should slowly shift the focus from REST to generic advice with examples of each tech. If that makes sense, then the title should not related to REST

Reply:
I've rephrased the titles to include both Swagger and GraphQL now, think it's the best for now